### PR TITLE
Adds multi-mind support to slimepeople

### LINKED ISSUE #34688: Slime Swapping with multiple minds sometimes swaps the wrong mind
[Round ID]: # 82223

[Testmerges]: # None

[Reproduction]: # As a slime person, create multiple duplicates, then put another player's brain in one. Attempting to switch will sometimes switch the other player.

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
tweak: Slimepeople can now host multiple minds in their body pool; occupied bodies will be marked as such in the bodyswap panel.
tweak: Slimepeople now retain 45% of their slime volume upon splitting instead of a fixed amount.
/:cl:

This sorta worked before, but buggily. Also cleaned up some code while i was there.

Fixes #34688

### CODE REVIEW

**Comment by coiax** [comment]
Is there any situation where you can have multiple minds sharing a body pool?

**Comment by XDTM** [comment]
Mindswap i guess, also according to a bug report someone managed to by inserting someone's brain into the body, which would involve the owner not opening the panel until the operation was completed. I might make it so bodies stay in the pool after death just so this can happen.

**Comment by coiax** [comment]
But you can't mindswap into a catatonic.

Although, during the Possessing Ghosts event, they CAN take control of a catatonic mob.

**Comment by XDTM** [comment]
With the last change i made, you can enter a slime pool by getting your brain implanted into a dead slimeperson body, then getting revived/cloned

**Comment by coiax** [comment]
You know, I think if you cut the brain out of a body, and put a new one in, the body is defibbable.

In fact, anything that just uses `death()` causes death by oxyloss I believe.

**Comment by XDTM** [comment]
It is indeed, i've done it a few times with monkeypeople back when you couldn't clone brains. Strange reagent also works without a time limit, and you can cure damage with synthflesh to make it usable.

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 1]
```diff
@@ -219,7 +221,7 @@
 	spare.domutcheck()
 	spare.Move(get_step(H.loc, pick(NORTH,SOUTH,EAST,WEST)))
 
-	H.blood_volume = BLOOD_VOLUME_SAFE
+	H.blood_volume *= 0.45
```
Comment (AnturK) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: This seems unrelated.

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm (hunk 1 above)
Comment (XDTM) [reply code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: I forgot about that, i used it for testing

I'll either pr it separately or document it if you want

Comment (AnturK) [reply code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: Please do either.

Comment (XDTM) [reply code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: Documented and kept in; it's not very related but i'd rather not make a whole new pr for a likely uncontroversial change

**Comment by uraniummeltdown** [comment]
could this mean the return of borers?
